packages/common/src/utils/formatting.ts

```diff
@@ -59,33 +59,72 @@ function valueIsNaN(value: any) {
     return Number.isNaN(Number(value));
 }
 
-function roundNumber(value: any, round: number | undefined): string {
-    if (round === undefined || round < 0) {
-        return `${value}`;
-    }
+function roundNumber(
```

Comment:
we should probably rename this function, as it does not only round, maybe `formatNumber` ?

Reply:
I think we can leave that for a future PR. I think we need to rethink how we can format numbers in a cleaner way.